packages/pds/tests/account.test.ts

```diff
@@ -168,6 +168,36 @@ describe('account', () => {
     ])
   })
 
+  it('allows administrative email updates', async () => {
+    await agent.api.com.atproto.admin.updateAccountEmail(
+      {
+        account: handle,
+        email: 'alice-new@test.com',
```

Comment:
Could toss some caps in here to test the normalization if we add that to the backend.

Reply:
👍